sql/2013/01/webp-adoption-by-wordpress-version.sql

```diff
@@ -0,0 +1,134 @@
+# HTTP Archive query to get WebP adoption by WordPress (major) version.
+#
+# WPP Research, Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See https://github.com/GoogleChromeLabs/wpp-research/pull/17
+
+SELECT
```

Comment:
Can you please format the entire query "correctly" for consistent syntax? Please use the "Format query" function of the BigQuery editor, see screenshot below:
<img width="353" alt="Screen Shot 2023-01-09 at 5 13 06 PM" src="https://user-images.githubusercontent.com/3531426/211439716-1ac229b3-cd67-4cb2-8d70-ed5adbbe24dc.png">

Reply:
Ha! I was formatting this using the "FORMAT" feature of a different tool I was using. I'll try running in the Big Query tool to see what changes.

Reply:
I reformatted this using the Big Query GCP console Format command and got slightly different results than before which I committed in [`605db7d` (#17)](https://github.com/GoogleChromeLabs/wpp-research/pull/17/commits/605db7d54cbced945bb9f977c62424364994252c)

Reply:
@adamsilverstein Looks like you formatted it correctly there, but then added a new file instead of renaming it in https://github.com/GoogleChromeLabs/wpp-research/pull/17/commits/9f24f895e606c7c2ea2f2045c7dc859b255b5d5a, again using the older format.